### PR TITLE
Users/mg/mcupload

### PR DESCRIPTION
**Pull request by madhurig**
- Allow build to succeed if symbols are not found
- Pass user-agent to Mobile Center calls
- Update formatting

### CODE REVIEW

File: Tasks/VSMobileCenterUpload/utils.ts [hunk 1]
```diff
@@ -6,15 +6,22 @@ import glob = require('glob');
...
         var workDir = tl.getVariable("System.DefaultWorkingDirectory");
         if (arePathEqual(p, workDir)) {
             // Path points to the source root, ignore it
             p = null;
         } else {
-            // will error and fail task if it doesn't exist.
-            tl.checkPath(p, name);
+            if (continueOnError) {
+                if (!tl.exist(p)) {
+                    tl.warning(`Failed to locate ${name} at ${p}`);
```
Comment (yacaovsnc) [inline Tasks/VSMobileCenterUpload/utils.ts]: Should we localize this warning?

File: Tasks/VSMobileCenterUpload/utils.ts [hunk 2]
```diff
@@ -6,15 +6,22 @@ import glob = require('glob');
```
Comment (yacaovsnc) [inline Tasks/VSMobileCenterUpload/utils.ts]: Do we still need this import if we moved to tl.glob?

**Comment by yacaovsnc** [comment]
LGTM.

File: Tasks/VSMobileCenterUpload/utils.ts (hunk 1 above)
Comment (madhurig) [inline Tasks/VSMobileCenterUpload/utils.ts]: :) Yes

**Pull request merged**